### PR TITLE
Feature/branding

### PR DESCRIPTION
**Pull request by mindblight**

### CODE REVIEW

File: src/core/styles/global/forms.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/*{*/
```
Comment (UnsungHero97) [inline src/core/styles/global/forms.css]: remove this

File: src/views/Main/styles.module.css [hunk 2]
```diff
@@ -34,7 +34,7 @@ body {
 }
 
 .appContent {
-  background-color: #fafbfc;
+  background-color: #ffffff;
```
Comment (UnsungHero97) [inline src/views/Main/styles.module.css]: :(

File: src/components/sidenav/SideNav.js [hunk 3]
```diff
@@ -19,16 +19,16 @@ class SideNav extends React.Component {
   getSideNavItemLayout = (route :string, text :string, faIconType :string) => {
 
     return (
-      <Link
-          to={`/${route}`}
-          activeClassName={styles.sideNavItemSelected}>
         <div className={styles.sideNavItem}>
+          <Link
```
Comment (UnsungHero97) [inline src/components/sidenav/SideNav.js]: are you sure this still works as before?

Comment (mindblight) [inline src/components/sidenav/SideNav.js]: Yep, it's easier to style this way.

File: src/core/styles/global/forms.css (hunk 1 above)
Comment (mindblight) [inline src/core/styles/global/forms.css]: +1

File: src/views/Main/styles.module.css (hunk 2 above)
Comment (mindblight) [inline src/views/Main/styles.module.css]: It actually contrasts pretty well with the sidebar :)

**Pull request merged**